doc/source/getting_started/intro_tutorials/air_quality_no2.rst

```diff
@@ -0,0 +1,24 @@
+:orhpan:
```

Comment:
I would have expected `:orphan:` to avoid [the build error](https://github.com/pandas-dev/pandas/pull/38735/checks?check_run_id=1615499945#step:6:2402):

```
air_quality_no2.rst: WARNING: document isn't included in any toctree
```

From [the Sphinx docs](https://www.sphinx-doc.org/en/master/usage/restructuredtext/field-lists.html#file-wide-metadatahttps://www.sphinx-doc.org/en/master/usage/restructuredtext/field-lists.html#file-wide-metadata),

> If [`:orphan:` is] set, warnings about this file not being included in any toctree will be suppressed.

Would love any suggestions.

Reply:
i think you need the `:orphan:` right underneath the include (e.g. this is a directive)

Reply:
It would help if I spelled `:orphan:` right 🤦 All good now!